tests/impl.cpp

```diff
@@ -723,6 +726,17 @@ bool validateSingleFloatPair(float a, float b)
                          // into account as well.
 }
 
+bool validateSingleDoublePair(double a, double b)
+{
+    const uint64_t *ia = (const uint64_t *) &a;
+    const uint64_t *ib = (const uint64_t *) &b;
+    return (*ia) == (*ib)
```

Comment:
It is not necessary to place ternary operator (`?`) here.

Reply:
Removed.